README.md

```diff
@@ -0,0 +1,31 @@
+# JuliaBoxTutorials
+
+This repository contains introductory tutorials for the [Julia language](http://julialang.org/) in the form of [Jupyter Notebooks](https://jupyter.org/about).
+
+## JuliaBox
+
+These tutorials are installed by default on the [JuliaBox](https://juliabox.com/) environment, where you can run Julia in your browser
+through a hosted notebook server. This service what these tutorials were built for!
+
+Go to https://juliabox.com/ and make an account to start running Julia in your browser.
+
+## Installing on your Computer
+
+You can also do these tutorials by installing `julia` on your computer, setting up Jupyter, and downloading this tutorial repository to your computer.
+If you're new to Julia, you can do that by following these steps:
+
+1. Download julia from https://julialang.org/downloads/ (download the latest "stable" version).
+   - Follow the instructions to install it on your computer.
+2. Install julia's Jupyter Notebooks integration: IJulia.jl
+   - Open julia, and you are presented with a "REPL" prompt. This is the main Julia interface. There, type this closing bracket
```

Comment:
How do I open julia?  Are we okay punting on this point?

Reply:
Mmm good point.
Do you think the changes in d376d0eef65594ee62bda1f9af68d5c624fb48c0 make this any clearer?

Reply:
I think so!